packages/dd-trace/src/appsec/iast/vulnerabilities-formatter/evidence-redaction/tokenizers/sql-regex-tokenizer.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+const iastLog = require('../../../iast-log')
+
+const STRING_LITERAL = /'(?:''|[^'])*'/
+const POSTGRESQL_ESCAPED_LITERAL = /\$([^$]*)\$.*?\$\1\$/
+const MYSQL_STRING_LITERAL = /"(?:\\"|[^"])*"|'(?:\\'|[^'])*'/
+const LINE_COMMENT = /--.*$/
+const BLOCK_COMMENT = /\/\*[\s\S]*\*\//
+const EXPONENT = /(?:E[-+]?\d+[fd]?)?/
+const INTEGER_NUMBER = /(?<!\w)\d+/
+const DECIMAL_NUMBER = /\d*\.\d+/
+const HEX_NUMBER = /x'[0-9a-f]+'|0x[0-9a-f]+/
+const BIN_NUMBER = /b'[0-9a-f]+'|0b[0-9a-f]+/
+const NUMERIC_LITERAL = new RegExp(
```

Comment:
Why are these all RegExp and not string literals if they are used in `new RegExp(xx)`?